app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= text_f f, :url, :size => "col-md-8", :help_block => _("e.g. https://ovirt.example.com/api"), :help_inline => documentation_button('5.2.7oVirt/RHEVNotes') %>
 <%= text_f f, :user, :help_block => _("e.g. admin@internal") %>
 <%= password_f f, :password, :unset => unset_password? %>
+<%= checkbox_f f, :filtered_api, :help_block => _("If you don't have admin permissions you can use slower filtered API, that works with user account."), :label => _('Filtered API') %>
```

Comment:
I'm not sure if it looks any better
![popover1](https://cloud.githubusercontent.com/assets/109773/13773604/6ab79766-ea9a-11e5-8e34-5ad55cf1dfee.png)

This is what we have for interfaces flag, also does not look great to me.
![popover2](https://cloud.githubusercontent.com/assets/109773/13773701/f86f5ab2-ea9a-11e5-9a93-8ecd0bc085f4.png)

I think I find this the best.
![popover3](https://cloud.githubusercontent.com/assets/109773/13773730/252b2964-ea9b-11e5-887f-b259330132f9.png)

Which one do you prefer?